packages/deprecated-component-library-helpers/styles/type.scss

```diff
@@ -94,9 +101,9 @@ $ca-ideal-locale-he-font-family: "Ideal Sans A", "Ideal Sans B", Tahoma,
   );
 }
 
-@mixin ca-type-ideal-small-bold($args...) {
-  @include ca-type-ideal-small;
-  font-weight: $ca-weight-semibold;
+@mixin ca-type-inter-small-bold($args...) {
```

Comment:
Could you duplicate this mixin too? Other wise this will have to be a breaking change

Reply:
Yes, that was a mistake, I have pushed a commit.

Reply:
fixed